### ISSUE TITLE
Bug in @use (idempotency)

### ISSUE THREAD

**Issue by najamelan** [connorskees/grass#49]
**Minimal Reproducible Example**:
```scss
// common.scss

$color: #fff;
```

```scss
// sub.scss

@use "common";

body
{
   font-color: common.$color;
}
```

```scss
// super.scss

@use "common";
@use "sub";

div
{
   font-color: common.$color;
}
```

**`grass` Output**:
```
Error: There's already a module with namespace "common".
  ╷
1 │ @use "common";
  │       ^^^^^^^
  ╵
./sub:1:7
```

**`dart-sass` Output**:
```css
body {
  font-color: #fff;
}

div {
  font-color: #fff;
}

/*# sourceMappingURL=super.css.map */
```

**Comment by connorskees** [comment]
Thanks for flagging! I think this might be related to #44. I'll see if I can find a fix for this tonight.

**Comment by connorskees** [comment]
This has been resolved in https://github.com/connorskees/grass/commit/d101a36f0ccf8bbd286fcbf13cff9b08e3d25363. I will update this thread again when the fix has been released to crates.io. Thanks for your report!

**Comment by connorskees** [comment]
Fix released in `0.10.7`
